### ISSUE TITLE
not play when manually change seekbar position (when remotly play encrypted video)

### ISSUE THREAD

**Issue by yaservaziri** [smmousavi8872/MMPlayer#4]
hi mohsen
i use your code to play an encrypted video remotly. it work and play but when manually change seekbar position then it dont play

**Comment by mason-doulabi** [comment]
Hi there,

This error probably refers to a malformed file position exception. After seeking, you should also use the modifyPosition() method.

To provide more accurate answer, please share you Error Log.

**Comment by ladeng** [comment]
@yaservaziri 
Have you solved the problem of not being able to seekto when playing encrypted videos? I've been stuck here recently!

**Comment by yaservaziri** [comment]
> @yaservaziri 
> Have you solved the problem of not being able to seekto when playing encrypted videos? I've been stuck here recently!

No. It just work for 10-15seconds ⏩. But it not work for backward or forward for more than minute. If you can solve, please contact me ( @yaservaziri )
Can you help us? @smmousavi8872

**Comment by ghost** [comment]
Was having the same issue with 'HttpEncryptedDataSourceFactory' (which has a typo in its DataSource class btw: 'AES/ECB/PCSK5-Padding' should be 'AES/ECB/PCSK5Padding'). 
Streaming worked but seeking backwards or to a position not yet loaded would crash the player.
So I migrated to 'AesCtrHttpEncryptedFileDataSourceFactory' with 'AES/CTR/NoPadding' and now everything works fine!
For reference I used the following code to encrypt my videos using .NET with the portable bouncy castle nuget package:

```
public static void AESCTRnoPadding(bool encryptMode, string input, string output, string key, string iv)
{
    IBufferedCipher cipher = CipherUtilities.GetCipher("AES/CTR/NoPadding");
    cipher.Init(encryptMode, new ParametersWithIV(ParameterUtilities.CreateKeyParameter("AES", Encoding.UTF8.GetBytes(key)), Encoding.UTF8.GetBytes(iv)));
    System.IO.File.WriteAllBytes(output, cipher.DoFinal(System.IO.File.ReadAllBytes(input)));
}
```
